app/gateways/http/account/create_test.go

```diff
@@ -48,38 +50,56 @@ func TestCreate(t *testing.T) {
 	})
 
 	t.Run("should return 400 and error when body has invalid fields", func(t *testing.T) {
+		body := CreateAccountRequest{
+			Name:   "Peter Park",
+			CPF:    "one two three",
+			Secret: "MySecret",
+		}
+		requestBody, _ := json.Marshal(body)
+		err := errors.New("invalid cpf")
+		handler := createFakeHandler(err)
+		request, _ := http.NewRequest(http.MethodPost, "/accounts", bytes.NewReader(requestBody))
+		response := httptest.NewRecorder()
+
+		http.HandlerFunc(handler.Create).ServeHTTP(response, request)
+
+		got := response.Body.String()
+		expected := `{"error":"invalid cpf"}`
+
+		assert.Equal(t, http.StatusBadRequest, response.Code)
+		assert.Equal(t, expected, got)
+		assert.Equal(t, http_server.JsonContentType, response.Header().Get("Content-Type"))
+	})
+
+	t.Run("should return 400 and error when body has invalid type fields", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("should return 400 and error when body has invalid field types", func(t *testing.T) {
```
I believe you've got a typo here, what do you think of this suggestion?

Reply:
My mistake! I will correct!